docs/docusaurus/docusaurus.config.js

```diff
@@ -136,6 +136,29 @@ module.exports = {
           to: '/docs/reference/api',
           position: 'right'
         },
+        {
+          type: 'dropdown',
+          label: 'Resources',
+          items: [
+            {
+              label: 'How to get support',
+              to: '/docs/resources/get_support'
+            },
+            {
+              label: 'Expectations Gallery',
```

Comment:
Lets make these labels sentence 
```suggestion
              label: 'Expectations gallery',
```